packages/eslint-plugin-tslint/tsconfig.json

```diff
@@ -5,6 +5,10 @@
     "rootDir": "."
   },
   "include": ["src", "tests"],
-  "exclude": ["tests/test-project", "tests/test-tslint-rules-directory"],
+  "exclude": [
+    "tests/fixture-project",
```

Comment:
Why was this not needed before?

Reply:
I think it is because there used to be a TSConfig in the tests folder which technically hid the files from this config?
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin-tslint/tests/tsconfig.json

I deleted this tsconfig because it's not how we do things, so the files got included, so now they need to be excluded.